### PR TITLE
change font, style fonts on pages

### PR DESCRIPTION
**Pull request by PawelKrawczyk96**

### CODE REVIEW

**Review by Pawel-Szydlo — COMMENTED** [review]
change padding-top

obraz

File: src/components/HomePageSections/Features/style.ts [hunk 1]
```diff
@@ -83,6 +83,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     ...typography.subtitle2,
     color: colors.navy.lightGrey,
     transition: 'transform 300ms linear',
+    position: 'relative',
```
Comment (awojciak) [inline src/components/HomePageSections/Features/style.ts:86]: why is this needed here?
